packages/client-core/src/common/services/MediaInstanceConnectionService.ts

```diff
@@ -90,62 +81,28 @@ export function useMediaInstance() {
   return mediaHostId.value ? mediaInstanceState[mediaHostId.value] : null
 }
 
-export const MediaInstanceConnectionServiceReceptor = (action) => {
-  const s = getMutableState(MediaInstanceState)
-  matches(action)
-    .when(MediaInstanceConnectionAction.serverProvisioned.matches, (action) => {
-      getMutableState(NetworkState).hostIds.media.set(action.instanceId)
-      const existingNetwork = getState(NetworkState).networks[action.instanceId]
-      if (!existingNetwork) {
-        addNetwork(initializeNetwork(action.instanceId, NetworkTopics.media))
-        return s.instances[action.instanceId].set({
-          ipAddress: action.ipAddress,
-          port: action.port,
-          channelId: action.channelId,
-          roomCode: action.roomCode,
-          videoEnabled: false,
-          provisioned: true,
-          readyToConnect: true,
-          connected: false,
-          connecting: false
-        })
-      }
-      return s
-    })
-    .when(MediaInstanceConnectionAction.serverConnecting.matches, (action) => {
-      return s.instances[action.instanceId].connecting.set(true)
-    })
-    .when(MediaInstanceConnectionAction.serverConnected.matches, (action) => {
-      s.joiningNewMediaChannel.set(false)
-      return s.instances[action.instanceId].merge({
-        connected: true,
-        connecting: false,
-        readyToConnect: false
-      })
-    })
-    .when(MediaInstanceConnectionAction.enableVideo.matches, (action) => {
-      return s.instances[action.instanceId].merge({
-        videoEnabled: action.enableVideo
-      })
-    })
-    .when(MediaInstanceConnectionAction.disconnect.matches, (action) => {
-      return s.instances[action.instanceId].set(none)
-    })
-    .when(MediaInstanceConnectionAction.changeActiveConnectionHostId.matches, (action) => {
-      const currentNetwork = s.instances[action.currentInstanceId].get({ noproxy: true })
-      const networkState = getMutableState(NetworkState)
-      const currentNework = getState(NetworkState).networks[action.currentInstanceId]
-      updateNetworkID(currentNework as SocketWebRTCClientNetwork, action.newInstanceId)
-      networkState.hostIds.media.set(action.newInstanceId as UserId)
-      s.instances.merge({ [action.newInstanceId]: currentNetwork })
-      s.instances[action.currentInstanceId].set(none)
-    })
-    .when(MediaInstanceConnectionAction.joiningNewMediaChannel.matches, () => {
-      return s.joiningNewMediaChannel.set(true)
+export const provisisionMediaInstanceServer = (
```

Comment:
`provisisionMediaInstanceServer` needs to be added to `MediaInstanceConnectionService` instead of as a top level export